### PR TITLE
Implement ticket-fees API in admin settings & add ticket-fee model

### LINKED ISSUE #528: Implement Ticket Fees API to admin/settings/ticket-fees
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Work to be done:
- Create ticket fees model
- Implement Ticket Fees API to admin/settings/ticket-fees

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Implements ticket-fees API in admin settings & add ticket-fee model

#### Changes proposed in this pull request:

- Implement ticket-fees API in admin settings & add ticket-fee model
- Add ticket-fee model
- Improve ticket-fees-form test

#### Deployment link 
http://cc96-opev-fe-omega.herokuapp.com/admin/settings/ticket-fees

Fixes #528

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review, had to add speaker model to application serializer because the tests started failing after updating the branch.

**Comment by CosmicCoder96** [comment]
@fossasia/open-event-frontend Please review

File: app/components/forms/admin/settings/ticket-fees-form.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const { Component } = Ember;
 export default Component.extend({
   actions: {
     submit() {
+      this.sendAction('save');
```
Comment (niranjan94) [inline app/components/forms/admin/settings/ticket-fees-form.js]: use `save` action directly in the template. No need this intermediate `submit` action here.

**Comment by CosmicCoder96** [comment]
@niranjan94 Done, please review!

**Pull request merged**